lib/s3routes/routesUtils.ts

```diff
@@ -10,6 +10,11 @@ import * as constants from '../constants';
 import DataWrapper from '../storage/data/DataWrapper';
 import StatsClient from '../metrics/StatsClient';
 
+export type ApiMethods = {
+        callApiMethod: CallApiMethod;
+        callPostObject: CallApiMethod;
```

Comment:
indent problem not caught up by linter

```suggestion
    callApiMethod: CallApiMethod;
    callPostObject: CallApiMethod;
```